units/provisioning_agent_deploy.go

```diff
@@ -298,6 +298,9 @@ func (j *agentDeployJob) startAgentOnRemote(ctx context.Context, settings *everg
 		fmt.Sprintf("--log_prefix='%s'", filepath.Join(hostObj.Distro.WorkDir, "agent")),
 		fmt.Sprintf("--working_directory='%s'", hostObj.Distro.WorkDir),
 		fmt.Sprintf("--logkeeper_url='%s'", settings.LoggerConfig.LogkeeperURL),
+		fmt.Sprintf("--s3_key='%s'", settings.Providers.AWS.Id),
+		fmt.Sprintf("--s3_secret='%s'", settings.Providers.AWS.Secret),
+		fmt.Sprintf("--s3_bucket='%s'", settings.Providers.AWS.Bucket),
```

Comment:
set these in environment variables, as this shows up in the output of ps, which we're piping to (in some cases) s3 which could be publicly visible.

Reply:
done